modules/social_features/social_group/social_group.module

```diff
@@ -2021,6 +2028,22 @@ function social_group_get_allowed_visibility_options_per_group_type($group_type_
  * Implements hook_views_query_alter().
  */
 function social_group_views_query_alter(ViewExecutable $view, QueryPluginBase $query) {
+  // User Groups view.
+  if ($view->id() == 'groups' && $view->getDisplay()->display['id'] == 'page_user_groups') {
+    // Change Join type if we want to see muted groups.
+    // @todo: Remove when will be fixed issue where exposed flag filter can't
```

Comment:
Please update this documentation with a better explanation of what to remove.

Reply:
Updated.